site/content/docs/5.3/components/badge.md

```diff
@@ -18,7 +18,7 @@ Badges scale to match the size of the immediate parent element by using relative
 <summary>See Bootstrap example(s) that are incompatible with Orange Design System.</summary>
 <br>
 {{< design-callout-alert >}}
-This component variant should not be used because it does not exist in the Orange Design System specifications.
```

Comment:
There are still 2 occurrences of "component variant" in the code in `badge.md` and `tables.md`. Shouldn't we get rid of them?